Sources/HaCWebsiteLib/Models/Link.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+/// Models a hyperlink
+struct Link {
```

Comment:
Is this actually being used anywhere in this PR?

Reply:
It is now! 😬